tests/examples_arguments_syntax/multiline_tooltip_standard.py

```diff
@@ -0,0 +1,52 @@
+"""
+Multi-Line Tooltip (Standard)
+=============================
+This example shows how to add a standard tooltip to the same chart 
+as in :ref:`gallery_multiline_tooltip`. You can find another example
+using this approach in the documentation on the :ref:`user-guide-pivot-transform` transformation.
+"""
+# category: interactive charts
+import altair as alt
+import pandas as pd
+import numpy as np
+
+np.random.seed(42)
+columns = ["A", "B", "C"]
+source = pd.DataFrame(
+    np.cumsum(np.random.randn(100, 3), 0).round(2),
+    columns=columns,
+    index=pd.RangeIndex(100, name="x"),
+)
+source = source.reset_index().melt("x", var_name="category", value_name="y")
+
+# The basic line
+line = (
+    alt.Chart(source)
+    .mark_line(interpolate="basis")
+    .encode(x="x:Q", y="y:Q", color="category:N")
+)
```

Comment:
I'm curious, do you find this more convenient than the approach we use elsewhere in the docs?

```
alt.Chart().mark_line(),encode(
    ...
)
```

I'm not suggesting you change it here, just curious about what you use on a day to day basis.

Reply:
Good point, they should be the same! I harmonised the code style, mainly adapting the new example to the old one so it's consistent with the approach we use in other places. I also slightly cleaned up the old one.

Personally, I just always use a code formatter (black or ruff). I love that I don't have to think about it and can get nicely formatted code with a hotkey or by saving a file :) The style I used before comes from ruff.